coreos-meta-translator/trans.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser, FileType
+from datetime import datetime
+import json
+import os
+
+
+def ensureDup(inp, out, inp_key, out_key):
+    if out.get(out_key, None) == None:
+        out[out_key] = inp.get(inp_key)
+        return out
+    if out.get(out_key) != inp.get(inp_key):
+        print("Input Files do not appear to be for the same release")
+        raise SystemExit(1)
+    return out
+
+def url_builder(stream, version, arch, path):
+    return f"https://fcos-builds.s3.amazonaws.com/prod/streams/{stream}/builds/{version}/{arch}/{path}"
```

Comment:
Can this point to the non-AWS URL for the bucket?

Reply:
Do we have a redirection URL to use? If so I can update here & in `plume`.

Reply:
You should be able to do just `s/fcos-builds.s3.amazonaws.com/builds.coreos.fedoraproject.org/`.